lib/capistrano/sumo/defaults.rb

```diff
@@ -1,14 +1,20 @@
 namespace :deploy do
   # compile and upload the assets
   after :updated, 'sumo:assets:put'
+  after :updated, 'sumo:migrations:prepare'
```

Comment:
I think it would be more logical to invoke this in the namespace itself instead of calling it here.